satpy/readers/avhrr_l1b_gaclac.py

```diff
@@ -286,16 +288,16 @@ def _get_qual_flags(self):
             self.qual_flags = self.reader.get_qual_flags()
         return self.qual_flags
 
-    def _get_angle(self, name):
-        """Get angle by name and buffer results."""
+    def _get_angle(self):
+        """Get angles and buffer results."""
         if self.angles is None:
             sat_azi, sat_zenith, sun_azi, sun_zenith, rel_azi = self.reader.get_angles()
             self.angles = {'sensor_zenith_angle': sat_zenith,
                            'sensor_azimuth_angle': sat_azi,
                            'solar_zenith_angle': sun_zenith,
                            'solar_azimuth_angle': sun_azi,
                            'sun_sensor_azimuth_difference_angle': rel_azi}
-        return self.angles[name]
+        return self.angles
```

Comment:
Shouldn't this be `self.angles[key.name]` ?

Reply:
Can be both, no? But I can change it.